test/config/gateway.config.yml

```diff
@@ -0,0 +1,39 @@
+http:
+  port: 8080
+admin:
+  port: 9876
+  hostname: localhost
+apiEndpoints:
+  api:
+    host: '*'
+serviceEndpoints:
+  httpbin:
+    url: 'https://httpbin.org'
+policies:
+  - proxy
+  - rewrite
+pipelines:
+  - name: basic
+    apiEndpoints:
+      - api
+    policies:
+      -
+        rewrite:
+          -
+            condition:
+              name: pathmatch
+              match: /gigino/:code
```

Comment:
Could you please check how variant with multi-segment match is working 
match /api/:route*
rewrite /api/v1/:route

incoming url  /api/users/nick
proxied to /api/v1/users/nick

Reply:
👍 I'll code all these as test cases.